App/features/contacts/selectors.ts

```diff
@@ -138,3 +139,9 @@ export const searchResults = (state: ContactsState) => {
 
   return sections
 }
+
+export const cafes = (contact: IContact) => {
+  return contact.peers.reduce((acc: ICafe[], peer) => {
+    return [...acc, ...peer.inboxes]
+  }, [])
+}
```

Comment:
Think you'll need to remove duplicates from this

Reply:
Oop, totally slipped my mind. Easy enough to change it to a set object.

Reply:
Fixed.